app/src/main/java/com/github/polybooks/core/Books.kt

```diff
@@ -24,6 +25,6 @@ data class Book(
         val edition : String?,
         val language : String?,
         val publisher : String?,
-        val publishDate : Date?,
-        val format : String?)
-
+        val publishDate : Timestamp?,
```

Comment:
Given that timestamps are a subclass of Date, is it really necessary to restrict to using Timestamps instead of date?

Reply:
Because they are not java timestamps but rather firebase timestamps. If we use this, then we can directly get the timestamp from database using getTimestamp (as we would for getNumber or getString)